staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -824,6 +838,14 @@ type KubeletConfiguration struct {
 	// Default: "10s"
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration `json:"shutdownGracePeriodCriticalPods,omitempty"`
+	// Reserved Memory specifies a comma separated list of memory reservations for NUMA nodes.
+	// Each configuration describes the reserved memory for desired memory types on a specific NUMA node.
+	// The Memory Manager validates whether total amount of reserved memory is identical to memory reserved by the Node Allocatable feature.
+	// The format is [{"numaNode": integer, "limits": {"type": "limit"}}, {"numaNode": integer, "limits": {"type": "limit"}}, ...]
+	// (e.g. [{"numaNode":integer, "limits":{"memory": "512Mi", "hugepages-2Mi": "512Mi"}}]
+	// Default: nil
+	// +optional
+	ReservedMemory []MemoryReservation `json:"reservedMemory,omitempty"`
```

Comment:
Where is the validation logic for this field?  I didn't see it in this PR

Reply:
The parsing logic placed under the flag `Set()` method - https://github.com/kubernetes/kubernetes/pull/95479/commits/6778d70f87374b0d25405727de083a8739125d40#diff-5ce184b1c6db0f7a5e25ba8283097c283ae66c084640be3cd4f17c71cb7ff111R163